pages/docs/open-source-roadmap/index.js

```diff
@@ -90,7 +90,7 @@ export default function DocsIndex() {
       },
       icon: ArrowUpRightIcon,
       comment:
-        "It doesn't stop there! You can review other people's changes and become a maintainer.",
+        "It doesn't stop there! You can become a maintainer. Have a close friend that's been raving to you about a project they are building? Consider asking them if you can help them maintain it. Present ways you can help maintain the project and when your available to do other duties.  If you want to build your own open source project, consider asking yourself the three W's: What am I building? Who is my audience? Why am I building this project?",
```

Comment:
In addition to my comment relating to where this text should go, please can you consider the messaging @CBID2 as I do think some more thought needs to go into this content. 

We do not want to give the impression that you can only become a maintainer for a project "a close friend" has - several people will feel excluded from this (I appreciate you were only using this as an example but best be more generic).  Please remember this roadmap is intended for those who are starting out in the Open Source world. 

Rather than "Present ways you can help maintain the project" - considering the audience for this roadmap, I would suggest giving actual practical tips of this as mentioned in the Issue.  You may wish to see the following I wrote for inspiration: [BioDrop Maintainers](https://www.biodrop.io/docs/maintainers).

You write about what to think about when considering whether to build your own Open Source project.  However as mentioned in the issue we discussed this section would include that you can become a maintainer in your own project.  Setting out why you should build a project might confuse people as it off the maintainer topic.

*Note: in the existing roadmap Open Source is capitalised.  Can you please make sure you reflect this as well.

Reply:
As discussed in the issue and the inline comment above I am making the necessary connection as this has not been done yet.


```suggestion
        "It doesn't stop there! You can become a maintainer.",
```